django_loci/static/django-loci/js/loci.js

```diff
@@ -147,7 +147,7 @@ django.jQuery(function ($) {
             });
             // name not relevant in mobile locations
             $name.parents('.form-row').hide();
-            $('.no-location').show();
+            if (!$geometryTextarea.val()) { $('.no-location').show(); }
```

Comment:
If `$geometryTextarea.val()` has a value, it's not hidden. Hence, map and the message appear together, this condition will help avoid that.